ohapi/cassettes/test_upload_file_remote_info_not_none_expired_access_token.yaml

```diff
@@ -0,0 +1,70 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: ['gzip, deflate']
+      Connection: [keep-alive]
+      User-Agent: [python-requests/2.9.1]
+    method: GET
+    uri: https://valid_url/
```

Comment:
There are a few uris that are also `https://valid_url/` - is this intended?

Reply:
As the function accepts any valid uri, For testing purposes I used SO and some other uris. However, since the cassettes should not be specific to any uri. Any valid uri will be fine. So for uniformity, the uris are changed to https://valid_url/.